templates/directives/report-verify-icon.html

```diff
@@ -0,0 +1,2 @@
+<svg ng-show="valid" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M23,7.14l-8.57,8.57-1.21,1.21a1.69,1.69,0,0,1-2.42,0L5.64,11.78A1.71,1.71,0,0,1,8.07,9.36L12,13.29l9.1-9.1A11.91,11.91,0,1,0,23.33,8,8.39,8.39,0,0,0,23,7.14Z" fill-rule="evenodd"/></g></svg>
+<svg ng-show="!valid" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M12,0A12,12,0,1,1,0,12,12,12,0,0,1,12,0Zm2.42,12,3.94-3.93a1.71,1.71,0,0,0-2.43-2.42L12,9.58,8.07,5.65A1.71,1.71,0,0,0,5.64,8.07L9.58,12,5.64,15.93a1.72,1.72,0,0,0,2.43,2.43L12,14.42l3.93,3.94a1.72,1.72,0,0,0,2.43-2.43Z" fill-rule="evenodd"/></g></svg>
```

Comment:
I think it would be better to use `ng-if` here not `ng-show` so it doesn't end up in the DOM. I wonder also if this would be better in two separate files so you could just `ng-include` into the page so there's no logic in them at all.

Reply:
I've put them in separate files and ng-included them in the template.